src/store/selectors/cartSelector.ts

```diff
@@ -2,7 +2,7 @@ import { createSelector } from "reselect";
 
 // Selector is a function which accepts the Redux state as an argument and returns the piece of the state that is requried.
 
-export const selectCart = (state: { cart: any }) => state.cart;
+export const selectCart = (state: { cartReducer: any }) => state.cartReducer;
```

Comment:
Can we use any specific type instead of `any`? `any `is a very powerful data type in typescript but at the same time it accepts everything which may cause errors in downstream code, could you please double-check once? thanks